src/effect/elliptical-mask.ts

```diff
@@ -49,14 +49,15 @@ export class EllipticalMask extends Base {
   apply (target: Movie | Visual, reltime: number): void {
     const ctx = target.cctx
     const canvas = target.canvas
-    const x = val(this, 'x', reltime)
-    const y = val(this, 'y', reltime)
-    const radiusX = val(this, 'radiusX', reltime)
-    const radiusY = val(this, 'radiusY', reltime)
-    const rotation = val(this, 'rotation', reltime)
-    const startAngle = val(this, 'startAngle', reltime)
-    const endAngle = val(this, 'endAngle', reltime)
-    const anticlockwise = val(this, 'anticlockwise', reltime)
+    this.currentTime = reltime
```

Comment:
No need to update currentTime

Reply:
Then it will be better to remove the parameters of reltime (bc linting warning), and that will require changes in many files. But in the long term, we'll have to look into it either way. Should i just temporarily fix it? and we can push these Eslint updates into tech debt.

Reply:
Oh, gotcha. Yeah, just remove line 52 and we can make another issue for deprecating reltime (because it's not really useful ever since currentTime was introduced).